Makefile

```diff
@@ -762,7 +762,7 @@ check test testshort testrace bench: gotestdashi
 
 # Run make testlogic/testccllogic to run all of the oss/ccl logic tests. Specify test files to run
 # with make testlogic FILES="foo bar".
-testlogic testccllogic: TESTS := $(if $(FILES),TestLogic$$//^$(subst $(space),$$|^,$(FILES))$$$(if $(SUBTESTS),/$(SUBTESTS)),TestLogic)
+testlogic testccllogic: TESTS := $(if $(FILES),Test.*Logic$$//^$(subst $(space),$$|^,$(FILES))$$$(if $(SUBTESTS),/$(SUBTESTS)),Test.*Logic)
```

Comment:
I renamed the test runner for ccl to `TestCCLLogic` so we can tell the difference in TC summaries. I had forgotten to adjust the makefile.